regexes/client/libraries.yml

```diff
@@ -566,3 +566,18 @@
   name: 'request'
   version: ''
   url: 'https://github.com/request/request'
+
+- regex: 'kiwi-tcms/([a-z\d\.A-Z]+)'
```

Comment:
```suggestion
- regex: 'kiwi-tcms/(\d+[\.\d]+)'
```

Reply:
You have to add a test user-agent to `Tests/Parser/Client/fixtures/library.yml`.